packages/config-service/tests/src/services/validationService.spec.ts

```diff
@@ -121,79 +121,74 @@ describe('ValidationService tests', async function () {
   describe('typeCasting', () => {
     it('should be able to use default value for missing env if default value is set', async () => {
       const castedEnvs = ValidationService.typeCasting({});
-      expect(castedEnvs).to.haveOwnProperty(GlobalConfig.ENTRIES.E2E_RELAY_HOST.envName);
-      expect(castedEnvs[GlobalConfig.ENTRIES.E2E_RELAY_HOST.envName]).to.equal(
-        GlobalConfig.ENTRIES.E2E_RELAY_HOST.defaultValue,
-      );
+      expect(castedEnvs).to.haveOwnProperty('E2E_RELAY_HOST');
+      expect(castedEnvs['E2E_RELAY_HOST']).to.equal(GlobalConfig.ENTRIES.E2E_RELAY_HOST.defaultValue);
     });
 
     it('should skip adding value if it is missing and there is no default value set', async () => {
       const castedEnvs = ValidationService.typeCasting({});
-      expect(castedEnvs).to.not.haveOwnProperty(GlobalConfig.ENTRIES.GH_ACCESS_TOKEN.envName);
-      expect(castedEnvs[GlobalConfig.ENTRIES.GH_ACCESS_TOKEN.envName]).to.be.undefined;
+      expect(castedEnvs).to.not.haveOwnProperty('GH_ACCESS_TOKEN');
+      expect(castedEnvs['GH_ACCESS_TOKEN']).to.be.undefined;
     });
 
     it('should to cast string type', async () => {
       const castedEnvs = ValidationService.typeCasting({
-        [GlobalConfig.ENTRIES.CHAIN_ID.envName]: '0x160c',
+        ['CHAIN_ID']: '0x160c',
```

Comment:
no need to use bracket notation now

```suggestion
        CHAIN_ID: '0x160c',
```

Reply:
Fixed.